internal/server/data/selectors.go

```diff
@@ -118,6 +119,22 @@ func ByUserID(userID uid.ID) SelectorFunc {
 	}
 }
 
+func ByPagination(pg *models.Pagination) SelectorFunc {
+	if pg == nil {
+		defaultpg := models.RequestToPagination(nil)
+		pg = &defaultpg
+	}
+
+	return func(db *gorm.DB) *gorm.DB {
+		// var count int64
+		// db.Count(&count)
+
+		pg.MaxPage = -999999 //TODO: get maxpage working
```

Comment:
This seems like it will set after the db query, so maybe applied to the result in `data.list` if pagination is used.